### DISCUSSION TITLE
motor driver module

### DISCUSSION THREAD

**Discussion by nliaudat** [deklaus/OpenValveControl#5] [Ideas]
Hi, 

To lower ther price, I would suggest to use another motor driver, perhaps 
- [Mini-L298N ](https://fr.aliexpress.com/item/1005001689987903.html)
- or L9110 chips directly on board

Regards

**Comment by deklaus** [comment]
Thanks for the tip, the price has not been my main focus so far, as the shipping costs for individual pieces for the project are similar anyway.
But watch out: The MX1508 is also often referred to as the L298. If I ever design the H-bridge on an assembled board, I will save more on the connectors than the cost of such a breakout board. I would also possibly realize the H-bridge with discrete FETs. This could have advantages when measuring the back EMF.

(Sorry for the long response time, I'm busy with other things at the moment. But I'll get back to it soon so that I can check off my to-do list before winter).
